KeenClient/KeenClient.h

```diff
@@ -203,54 +203,56 @@ typedef NSDictionary* (^KeenGlobalPropertiesBlock)(NSString *eventCollection);
 /**
  Call this to indiscriminately delete all events queued for sending.
  */
-+ (void)clearAllEvents;
++ (void)clearAllEvents DEPRECATED_MSG_ATTRIBUTE("Class method clearAllEvents is deprecated. Use instance method instead.");
```

Comment:
That's the idea. I marked as deprecated anything in this change that seemed like it obviously should go, but couldn't because it would require a major version revision.